data-table/table.js

```diff
@@ -43,6 +144,47 @@ function createEntryCell(indexString) {
      */
 }
 
+function cellIndexToElementId(rowIndex, colIndex) {
+    return "_row_" + rowIndex + "_and_col_" + colIndex + "_";
+}
+
+function createSubDivs(config) {
+    let entryBoxDiv = document.createElement("div");
+    entryBoxDiv.id = config.tableIds.entryBoxDivId;
+
+    let tableDiv = document.createElement("div");
+    tableDiv.id = config.tableIds.tableDivId;
+
+    // FIX: Delete this!
```

Comment:
[nit] FYI: "FIXME" and "TODO" often have special highlighting in many editors, so consider using one of those instead of just "FIX"

Reply:
Good to know!